src/app/payment/paymentMethod/DigitalRiverPaymentMethod.tsx

```diff
@@ -21,7 +21,31 @@ const DigitalRiverPaymentMethod: FunctionComponent<DigitalRiverPaymentMethodProp
     ...rest
 }) => {
     const { setSubmitted } = useContext(FormContext);
-    const containerId = `${rest.method}-component-field`;
+    const containerId = `${rest.method.id}-component-field`;
+    let disabledPaymentMethodsFromSettings: string[] = [];
+    const { initializationData } = rest.method;
+    if (initializationData) {
+        const { disabledPaymentMethods } = initializationData;
+
+        if (disabledPaymentMethods) {
+            disabledPaymentMethodsFromSettings = disabledPaymentMethods;
+        }
+    }
+
+    const disabledPaymentMethodsData = [
+        ...disabledPaymentMethodsFromSettings,
+        'alipay',
```

Comment:
Why are some disabled methods coming from the API (`rest.method.initializationData.disabledPaymentMethods`) 

while others are hardcoded in here in the view? (`disabledPaymentMethodsData`)

I.e. Why don't they all come from the API?

Reply:
The reason is: 
The list that is hardcoded is the apms those are out of scope. 
The `rest.method.initializationData.disabledPaymentMethods` are the apms that are supported and have checkboxes in DR settings page: 
![image](https://user-images.githubusercontent.com/35146660/120851430-6ea6c700-c53e-11eb-9ef0-9678e8c55b1b.png)

Reply:
Why not hardcode them in the BE?

It would help us keep a more singular source of truth for settings.
 
It would also allow us to simplify this code and stop settings from being kept forever in branches of checkout.

Reply:
Ok I'll move this settings to the BE